Index: CALPUVTT.for
===================================================================
--- CALPUVTT.for	(revision 3544)
+++ CALPUVTT.for	(working copy)
@@ -42,12 +42,12 @@
       DIMENSION QCHANUT(NCHANM),QCHANVT(NCHANM)
 
       PARAMETER (LLCM=200)
-      REAL,STATIC :: CCW1(LLCM),CCE1(LLCM),CCN1(LLCM),CCS1(LLCM)
-      REAL,STATIC :: CCC1(LLCM)
+      REAL,SAVE :: CCW1(LLCM),CCE1(LLCM),CCN1(LLCM),CCS1(LLCM)
+      REAL,SAVE :: CCC1(LLCM)
 
-      REAL,STATIC :: UHDY1ET(LLCM)
-      REAL,STATIC :: VHDX1ET(LLCM)
-      REAL,STATIC :: H1PT(LLCM)
+      REAL,SAVE :: UHDY1ET(LLCM)
+      REAL,SAVE :: VHDX1ET(LLCM)
+      REAL,SAVE :: H1PT(LLCM)
 
       REAL*8   DTMP
       REAL*4   EPSILON

