Index: EEXPOUT.for
===================================================================
--- EEXPOUT.for	(revision 3544)
+++ EEXPOUT.for	(working copy)
@@ -63,10 +63,10 @@
 C **  INITIAL CALL
       IF(JSEXPLORER.EQ.1)THEN
         OPEN(95,FILE='EE_WC.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
         CLOSE(95,STATUS='DELETE')
         OPEN(95,FILE='EE_WC.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
         VER=106
         WRITE(95)VER
         WRITE(95)ISTRAN(1),ISTRAN(2),ISTRAN(3),ISTRAN(4)
@@ -80,10 +80,10 @@
 
         IF(ISTRAN(6).GT.0.AND.NSEDFLUME.GT.0)THEN
           OPEN(95,FILE='EE_SEDZLJ.OUT',STATUS='UNKNOWN',
-     &           ACCESS='SEQUENTIAL',FORM='BINARY')
+     &           ACCESS='STREAM')
           CLOSE(95,STATUS='DELETE')
           OPEN(95,FILE='EE_SEDZLJ.OUT',STATUS='UNKNOWN',
-     &           ACCESS='SEQUENTIAL',FORM='BINARY')
+     &           ACCESS='STREAM')
           VER=100
           WRITE(95)VER
           WRITE(95)ITBM,NSICM 
@@ -93,10 +93,10 @@
         IF(ISBEXP.GE.1)THEN
           IF(ISTRAN(6).GE.1.OR.ISTRAN(7).GE.1)THEN
             OPEN(10,FILE='EE_BED.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
             CLOSE(10,STATUS='DELETE')
             OPEN(10,FILE='EE_BED.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
             VER=106
             WRITE(10)VER
             WRITE(10)ISTRAN(1),ISTRAN(2),ISTRAN(3),ISTRAN(4)
@@ -111,10 +111,10 @@
 
         IF(ISTRAN(8).GT.0)THEN
           OPEN(95,FILE='EE_WQ.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
           CLOSE(95,STATUS='DELETE')
           OPEN(95,FILE='EE_WQ.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
           VER=100
           WRITE(95)VER
           WRITE(95)NWQV
@@ -129,10 +129,10 @@
           ! *** SAVE SEDIMENT DIAGENESIS RESULTS
           IF(ISSDBIN.LT.0)THEN
             OPEN(95,FILE='EE_SD.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
             CLOSE(95,STATUS='DELETE')
             OPEN(95,FILE='EE_SD.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
             VER=100
             WRITE(95)VER
             WRITE(95)NACTIVE
@@ -156,7 +156,7 @@
 
       IF(ISSPH(8).GE.1)THEN
         OPEN(95,FILE='EE_WC.OUT',STATUS='OLD',
-     &         POSITION='APPEND',FORM='BINARY')
+     &         POSITION='APPEND',ACCESS='STREAM')
         WRITE(95)EETIME,NACTIVE
         DO L=2,LA
           N1=KBT(L)
@@ -217,7 +217,7 @@
       ! *** OUTPUT THE SEDZLJ VARIABLES
       IF(ISTRAN(6).GT.0.AND.NSEDFLUME.GT.0)THEN
         OPEN(95,FILE='EE_SEDZLJ.OUT',STATUS='OLD',
-     &         POSITION='APPEND',FORM='BINARY')
+     &         POSITION='APPEND',ACCESS='STREAM')
      
         WRITE(95)EETIME,NACTIVE
 
@@ -248,7 +248,7 @@
       IF(ISBEXP.GE.1)THEN
         IF(ISTRAN(6).GE.1.OR.ISTRAN(7).GE.1.AND.KB.GT.1)THEN
           OPEN(10,FILE='EE_BED.OUT',STATUS='UNKNOWN',POSITION='APPEND'
-     &           ,FORM='BINARY')
+     &           ,ACCESS='STREAM')
           WRITE(10)EETIME,NACTIVE
           DO L=2,LA
             WRITE(10)KBT(L)
@@ -286,7 +286,7 @@
           OPEN(95,FILE='EE_ARRAYS.OUT',STATUS='UNKNOWN')
 	    CLOSE(95,STATUS='DELETE')
           OPEN(95,FILE='EE_ARRAYS.OUT',STATUS='UNKNOWN',
-     &         ACCESS='SEQUENTIAL',FORM='BINARY')
+     &         ACCESS='STREAM')
           VER=100
           WRITE(95)VER
           WRITE(95)3    ! # OF TIME VARYING ARRAYS
@@ -334,7 +334,7 @@
 
         ELSE
           OPEN(95,FILE='EE_ARRAYS.OUT',STATUS='UNKNOWN',
-     &            POSITION='APPEND',FORM='BINARY')
+     &            POSITION='APPEND',ACCESS='STREAM')
         ENDIF
 
         IF(.TRUE.)THEN
@@ -537,7 +537,7 @@
         ! 20) TAM - total active metal
         ! 21) FCB - fecal coliform bacteria
         OPEN(95,FILE='EE_WQ.OUT',STATUS='UNKNOWN',POSITION='APPEND',
-     &          FORM='BINARY')
+     &          ACCESS='STREAM')
         WRITE(95)EETIME
         DO L=2,LA
           DO K=1,KC
@@ -558,7 +558,7 @@
           NSEDSTEPS=NSEDSTEPS+1
           IF(NSEDSTEPS.GE.ABS(ISSDBIN).OR.JSEXPLORER.EQ.1)THEN
             OPEN(95,FILE='EE_SD.OUT',STATUS='UNKNOWN',POSITION='APPEND',
-     &           FORM='BINARY')
+     &           ACCESS='STREAM')
             WRITE(95)EETIME
             DO L=2,LA
 

